LayoutTests/fast/forms/autofocus-input-css-style-change.html

```diff
@@ -11,11 +11,16 @@
 <script>
 if (window.testRunner) {
     testRunner.dumpAsText();
+    testRunner.waitUntilDone();
 }
 
 requestAnimationFrame(()=>{
     if (getComputedStyle(test, null).getPropertyValue('background-color') == "rgb(0, 128, 0)")
         result.innerHTML = "PASS";
+    else
+        result.innerHTML = 'FAIL - ' + getComputedStyle(test, null).getPropertyValue('background-color');
```

Comment:
Could store `getComputedStyle(test, null).getPropertyValue('background-color')` in a local variable since it's called twice, but not a big deal.

Reply:
Oh, true. Will do that.